.rustfmt.toml

```diff
@@ -0,0 +1,15 @@
+edition                     = "2024"
+unstable_features           = true
```

Comment:
you can enable unstable features here?  TIL, this is indeed much better!

Reply:
Yeah, it forces you to call with `cargo +nightly fmt` or use the nightly toolchain directly

Reply:
yeah, also discovered this.  This is a pretty big downside to be fair since now everyone will run into errors with this as it no longer works correctly with normal editor integrations.  In the main iroh repo we hooked up an external `cargo make format` command that can format it fine with stable rust without giving errors.  The benefit of that approach is that by default the formatting is close enough so that rust-analyzer etc will do the right thing and folks don't need to be aware of this.  Only very occasionally will you run into CI complaining about this and having to manually run `cargo make format`.  I appreciate you still have to install a new tool in that case.